pkg/asset/rhcos/image.go

```diff
@@ -171,9 +171,13 @@ func osImage(config *types.InstallConfig) (string, error) {
 			return config.Platform.PowerVS.ClusterOSImage, nil
 		}
 
-		if a, ok := streamArch.Artifacts["powervs"]; ok {
-			return rhcos.FindArtifactURL(a)
+		if streamArch.Images.PowerVS != nil {
+			vpcRegion := rhcos.PowerVSRegions[config.Platform.PowerVS.Region].VPCRegion
+			img := streamArch.Images.PowerVS.Regions[vpcRegion]
+			logrus.Debug("PowerVS Using Image ", img.Object)
```

Comment:
nit: can you change this from `PowerVS` to `Power VS`? And not capitalize `Using` or `Image`?